jerry-core/jerry.c

```diff
@@ -946,35 +954,29 @@ jerry_foreach_object_field (jerry_object_t *object_p, /**< object */
  *      if value was retrieved successfully, it should be freed
  *      with jerry_release_value just when it becomes unnecessary.
  *
- * @return true, if field value was retrieved successfully, i.e. upon the call:
- *                - there is field with specified name in the object;
- *         false - otherwise.
+ * @return ecma value
```

Comment:
In some places it's `ecma value` and in other places it's `jerry value`. I think the naming should be consistent.
Also, these `@return`s could be a little bit more descriptive in my opinion. I get that this is a simple getter function, but still.